src/main/java/com/autotune/database/table/lm/KruizeBulkJob.java

```diff
@@ -40,8 +40,8 @@
  */
 @Entity
 @Table(name = "kruize_bulkjobs")
-public class BulkJob {
-    private static final Logger LOGGER = LoggerFactory.getLogger(BulkJob.class);
+public class KruizeBulkJob {
```

Comment:
Should be `KruizeBulkJobEntry` to match the naming convention of the remaining set of tables